block-storage/src/main/scala/coop/rchain/blockstorage/dag/BlockDagStorage.scala

```diff
@@ -19,6 +20,8 @@ trait BlockDagStorage[F[_]] {
   ): F[Unit]
   def lookup(blockHash: BlockHash): F[Option[BlockMetadata]]
   def lookupByDeployId(blockHash: DeployId): F[Option[BlockHash]]
+  def addDeploy(d: Signed[DeployData]): F[Unit]
+  def pooled: F[Map[DeployId, Signed[DeployData]]]
```

Comment:
Please put _deploy_ in the name of this function.

Reply:
https://github.com/rchain/rchain/pull/3673/commits/3e8d3da8f2ea9f7c5fb8a030ebbc4cc203006dfa